apis/droplet_handler_test.go

```diff
@@ -0,0 +1,195 @@
+package apis_test
+
+import (
+	"errors"
+	"net/http"
+	"net/http/httptest"
+
+	"code.cloudfoundry.org/cf-k8s-api/repositories"
+
+	. "code.cloudfoundry.org/cf-k8s-api/apis"
+	"code.cloudfoundry.org/cf-k8s-api/apis/fake"
+	"github.com/gorilla/mux"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/client-go/rest"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var _ = Describe("DropletHandler", func() {
+	const (
+		testDropletHandlerLoggerName = "TestDropletHandler"
+	)
+	Describe("the GET /v3/droplet/:guid endpoint", func() {
```

Comment:
It's a minor thing, but I find it hard to read test files where there's no newlines between sibling `It`, `When`, etc. statements, like in this file. You don't need to change it for me, but I'll definitely add them if I'm ever working in this file. Maybe I should start a team discussion about this, since I know you're not the only one who leaves out the newlines and I may be in the minority here.

Reply:
Have addressed it. I don't have a strong inclination. I would like to hear from others 👍🏽